src/Components/Form.jsx

```diff
@@ -0,0 +1,163 @@
+import React, { useState } from "react";
+import styled from "styled-components";
+import { ImInfo } from "react-icons/im";
+function Form() {
+  const [breed, setBreed] = useState("");
+  const [weight, setWeight] = useState("");
+  const [height, setHeight] = useState("");
+  const [isInfo, setIsInfo] = useState(false);
+  const [heightUnits, setHeightUnits] = useState("in");
+  const [weightUnits, setWeightUnits] = useState("lb");
+
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    console.log(breed, weight, height, heightUnits, weightUnits);
+  };
+
+  const toggleWeight = (value) => {
```

Comment:
if this is a setter method, can you rename from toggle to something like `setWeight` instead? Toggle is generally used for a method that sets a boolean to be the opposite of whatever it's currently set to

Reply:
ill change